src/utils.jl

```diff
@@ -63,6 +92,31 @@ function postprocess_chunks(chunks::AbstractVector{<:AbstractString}, sources::A
     end
 
     return chunks, sources
+end
+
+"""
+    function remove_urls_from_index(index_path::AbstractString, prefix_urls=Vector{<:AbstractString})
+
+Remove chunks and sources corresponding to URLs starting with `prefix_urls` 
+"""
+function remove_urls_from_index(index_path::AbstractString, prefix_urls=Vector{<:AbstractString})
```

Comment:
Out of curiosity, what URLs do we tend to remove? I haven't noticed this before

Reply:
If the user accidentally includes a URL that the user might want to remove later from the index. I'd done this by including flux and makie in sciml. Also thought of putting it in if the crawler crawls some links which are not needed